config/samples/full.yaml

```diff
@@ -28,14 +28,14 @@ spec:
 
   cliServices:
     # Overrides the setting for version on this component
-    version: "0.2.0-rc.1"
+    version: "0.2.0"
 
     # Overrides the image as a separate repository or tag
     repository: kabanero/kabanero-command-line-services
-    tag: "v0.2.0rc1"
+    tag: "0.2.0"
 
     # Overrides the image uri
-    image: kabanero/kabanero-command-line-service:v0.2.0rc1
+    image: kabanero/kabanero-command-line-service:0.2.0
```

Comment:
Noticed that the repo names are different between repository: and image:. I think it needs to be: kabanero/kabanero-command-line-services